src/iso9660.rs

```diff
@@ -567,8 +579,23 @@ mod tests {
         IsoFs::from_file(iso_file).unwrap()
     }
 
+    fn open_truncated_iso() -> Result<IsoFs, Error> {
```

Comment:
Unlike `open_iso()`, this doesn't need to be a helper function, since it's only needed by one test.  So let's make this function a `#[test]` and do the `unwrap_err()` and comparison here, and drop the `assert_eq!` from `test_primary_volume_descriptor`.